dbms/src/Server/Server.cpp

```diff
@@ -1094,12 +1090,21 @@ int Server::main(const std::vector<std::string> & /*args*/)
     /// Initialize main config reloader.
     auto main_config_reloader = std::make_unique<ConfigReloader>(
         config_path,
-        [&](ConfigurationPtr config) {
+        [&](ConfigurationPtr config, bool is_new_tls_cert) {
+            LOG_DEBUG(log, "run main config reloader");
             buildLoggers(*config);
             global_context->setMacros(std::make_unique<Macros>(*config, "macros"));
             global_context->getTMTContext().reloadConfig(*config);
             global_context->getIORateLimiter().updateConfig(*config);
             global_context->reloadDeltaTreeConfig(*config);
+            bool updated = global_context->getSecurityConfig()->update(*config);
```

Comment:
I think we don't need to pass `is_new_tls_cert`, `global_context->getSecurityConfig()->update(*config);` can return true if tls path or tls file is changed

Reply:
> I think we don't need to pass `is_new_tls_cert`, `global_context->getSecurityConfig()->update(*config);` can return true if tls path or tls file is changed

Yes, I choose to keep it because I want a UT of update.
I will find a better way to write the UT

Reply:
If we don't pass the `is_new_tls_cert`, the `update()` will check whether the tls file changed twice.